pymongo/collection.py

```diff
@@ -115,6 +115,13 @@ def __init__(self, database, name, create=False, codec_options=None,
           - `**kwargs` (optional): additional keyword arguments will
             be passed as options for the create collection command
 
+        .. versionchanged:: 4.0
+           Removed the reindex, map_reduce, inline_map_reduce,
+           parallel_scan, initialize_unordered_bulk_op,
+           initialize_ordered_bulk_op, group, count, insert, save,
+           update, remove, find_and_modify, and ensure_index methods. See the
+           :ref:`pymongo4-migration-guide`.
```

Comment:
Perfect, can you make a similar change for MongoClient and Database?

Reply:
Done.